ts/sagas/messages/watchThirdPartyMessageSaga.ts

```diff
@@ -59,7 +59,9 @@ function* getThirdPartyMessage(
         loadThirdPartyMessage.success({ id, content: thirdPartyMessage })
       );
     } else {
-      const reason = `Response status ${result.right.status}`;
+      const reason = `Response status ${result.right.status} - ${
+        result.right.value?.detail || "UNKNOWN"
```

Comment:
What about `no "detail" field provided` instead of "UNKNOWN"?

Reply:
Applied https://github.com/pagopa/io-app/pull/5318/commits/7939171193c40283b703d5b4fe8df5b4d59577bc